packages/babel-code-frame/src/index.js

```diff
@@ -7,16 +7,18 @@ import chalk from "chalk";
  */
 
 let defs = {
-  string:     chalk.red,
-  punctuator: chalk.bold,
-  curly:      chalk.green,
-  parens:     chalk.blue.bold,
-  square:     chalk.yellow,
-  keyword:    chalk.cyan,
-  number:     chalk.magenta,
-  regex:      chalk.magenta,
-  comment:    chalk.grey,
-  invalid:    chalk.inverse
+  keyword:     chalk.yellow.bold,
+  capitalized: chalk.yellow,
+  jsx_tag:     chalk.yellow.dim,
+  punctuator:  chalk.yellow.dim,
+  // bracket:  intentionally omitted.
+  number:      chalk.yellow,
+  string:      chalk.yellow,
+  regex:       chalk.yellow,
+  comment:     chalk.grey,
+  invalid:     chalk.white.bgRed.bold,
+  gutter:      chalk.grey,
+  marker:      chalk.red.bold,
```

Comment:
Anyone want to play around with these colors? All you need to do is edit this block of code!

Quick-start:

1. `git clone https://github.com/babel/babel.git`
2. `make bootstrap`
3. Edit `packages/babel-code-frame/lib/index.js`.
4. Try it out with `node test.js`.
5. Post your changes and a screen shot here.

```
// test.js
const codeFrame = require('./packages/babel-code-frame/lib/')

const lines = `
class MyComponent extends React.Component {
  render() {
    const format = (title) => title.replace(/,$/, '') + 1337;
    return (
      <h1 className="test">
        {format(this.props.title)}
      </h1>
    );
  } // comment
} # invalid
`.trim()
console.log(codeFrame(lines, 6, 10, {highlightCode: true}))
```